vm_operator/config.yaml

```diff
@@ -0,0 +1,32 @@
+options:
```

Comment:
I wouldn't include this much config - it might sound odd but you want to be exposing *as little* config as possible to keep the operator code simple. 

I'd consider dropping:

- `app-host`
- `app-jwt-secret` - this is probably better autogenerated?
- `app-name`
- `app-port` - unless you're thinking of co-locating this unit on machines with other services on 443?

Both the postgres options need to be removed - this should be handled by relations to postgres.

With squid, it's worth asking IS, as I believe you should also be able to get this from a relation, though I could be wrong here.

Reply:
Following up - some of these don't actually seem to be used at all in the code.

I'd recommend the following:

- `env` - keep this, with some code to validate the entry as either `prod` or `dev`
- `log_level` - again with some validation of the valid options